nbgrader/docs/source/configuration/jupyterhub/jupyterhub_config.py

```diff
@@ -7,3 +7,24 @@
 c.Authenticator.whitelist = [
     "instructor1", "instructor2", "student1", "student2", "student3"
 ]
+
+# Add users here that are allowed to access the formgrader. NOTE: this option
+# strictly adds users. If you want to revoke formgrader access after running
+# JupyterHub, you will need to use the JupyterHub API to remove that user
+# from the formgrader group.
+c.JupyterHub.load_groups = {
+    'formgrader': ['instructor1', 'instructor2']
```

Comment:
I found it a little confusing to have the group name be formgrade because that name is used elsewhere (the name of the service). In practice I name the groups by the course. So maybe something generic but more descriptive such as `course1`?